joomla/joomla-cms
# [5.0] schemaorg fix schema forms

### Summary of Changes

- fix option value
- add missing language keys

### Testing Instructions

- enable debug mode and check form labels
- test fields `abridged` and `bookFormat` of book type (syntax error in form xml)

### Actual result BEFORE applying this Pull Request

![image](https://github.com/joomla/joomla-cms/assets/66922325/f09db569-390a-4507-a88d-e5bf76fb2b4b)

values for `abridged` and `bookFormat` are missing in schemaorg output

### Expected result AFTER applying this Pull Request

![image](https://github.com/joomla/joomla-cms/assets/66922325/6ef84846-e485-4dde-bc6c-f2f9dabbba50)

values for `abridged` and `bookFormat` are available in schemaorg output

### Link to documentations
Please select:
